browser_crawler.gemspec

```diff
@@ -34,9 +34,9 @@ Gem::Specification.new do |spec|
   spec.add_dependency "chromedriver-helper", '~> 2.1', '>= 2.1.0'
   spec.add_dependency "selenium-webdriver", '~> 3.141', '>= 3.141.0'
 
-
   spec.add_development_dependency "bundler", "~> 1.17.2", '>= 1.17.2'
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "rspec", "~> 3.0"
-  spec.add_development_dependency "pry-byebug", "~> 3.5.1", '>= 3.5.1'
```

Comment:
what is the reason for moving from pry-byebug to byebug? I think @DimaSamodurov  likes pry more, and that was the reason he added it

Reply:
I reverted this.